Add sensor platform documentation to goalzero (#17638)
home-assistant/home-assistant.github.io: source/_integrations/goalzero.markdown

- **12V Port Status**: Indicates if the 12V power port is currently on.
- **USB Port Status**: Indicates if the USB power port is currently on.
- **AC Port Status**: Indicates if the AC power port is currently on.

The following sensors will also be created:

- **Watts In**: Shows the watt input.
- **Amps In**: Shows the ampere input. (disabled by default)
- **Watts Out**: Shows the watt output.
- **Amps Out**: Shows the ampere output. (disabled by default)

> - **WH Out**: Shows the watt hour output. (disabled by default)

Reviewer: ```suggestion
- **WH Out**: Shows the watt-hour output. (disabled by default)
```

> - **WH Stored**: Shows the current watt hours stored.

Reviewer: ```suggestion
- **WH Stored**: Shows the current watt-hours stored.
```

> - **volts**: Shows the voltage of the battery. (disabled by default)

Reviewer: ```suggestion
- **Volts**: Shows the voltage of the battery. (disabled by default)
```

- **State of Charge Percent**: Shows the current battery charge percentage.
- **Temperature**: Shows the battery temperature.
- **Time to Empty/Full**: Shows the time left in minutes to empty or full depending on current usage. Shows -1 when there is no battery activity.

> - **Wifi Strength**: Shows the Wifi strength the device has with the connected access point.

Reviewer: ```suggestion
- **Wi-Fi Strength**: Shows the Wi-Fi strength the device has with the connected access point.
```
